mypy.ini

```diff
@@ -34,3 +34,8 @@ ignore_missing_imports = True
 # - setuptools._vendor.packaging._manylinux: Mypy issue, this vendored module is already excluded!
 [mypy-pkg_resources.tests.*,setuptools._vendor.packaging._manylinux,setuptools.config._validate_pyproject.*]
 disable_error_code = import-not-found
+
+# - The unused-ignore comment in setuptools.config._validate_pyproject.* is probably evaluated differently
+#   in different versions of Python. Also, this file should already be ignored...
+[mypy-setuptools.config._validate_pyproject.*]
+disable_error_code = unused-ignore
```

Comment:
> Add yet another exception in mypy.ini (¿shouldn't *._validate_pyproject.* already be excluded?)

Yes but it's a different error that's ignored. I'd suggest doing something like you did in https://github.com/pypa/setuptools/pull/4330/files#diff-6f2d4ba9ca9a357d31014946667b7bed1bfdbc6d2530afc77778fa0a36bee457R24-R29

```suggestion
# Even when excluding vendored/generated modules, there might be problems: https://github.com/python/mypy/issues/11936#issuecomment-1466764006
[mypy-pkg_resources.tests.*,setuptools._vendor.packaging._manylinux,setuptools.config._validate_pyproject.*]
follow_imports = silent
# silent => ignore errors when following imports
```